index.html

```diff
@@ -0,0 +1,101 @@
+    <link href="style.css" rel="stylesheet"/>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet"/>
+
+</head>
+<body>
+    <header>
+        <nav class="first-navigation">
+            <ul class="first-ul">
+                <li><a href="#"><i class="fa fa-facebook"></i></a></li>
+                <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Log out</a></li>
+            </ul>
+        </nav>
+        <nav class="sec-nav">
+            <span class="menu"><i class="fas fa-bars" id="menu"></i></span>  
+                <img class="logo" src="img/MyLogo.png" alt="logo">
+                    <ul class="sec-ul"  id="mobile-menu">
+                        <li><i class="fas fa-times close-btn" id="close-btn" ></i></li>
+                        <li><a href="/about.html" class="mobile-a">About</a></li>
+                        <li><a href="/index.html" class="mobile-a">Program</a></li>
+                        <li><a href="about.html" class="mobile-a">Join</a></li>
+                        <li><a href="#" class="mobile-a">Sponsor</a></li>
+                        <li><a href="#" class="button mobile-a">CC Campaign</a></li>
+                    </ul>
+        </nav>
+    </header>
+<main class="main-container">
+    <h1 class="first-h1">Hello! Welcome!!!</h1>
+        <h1 class="sec-h1">Meanwood Farmers Cooperative Conference</h1>
+            <p class="page-par">We take pride in helping out farmers get farming inputs on time in readiness for the farming season. Food security for our nation is our top priority
+            </p>
+            <p class="bold">
+           (MON)~(FRI) FROM 08:00am to 5:00pm
+            </p>
+            <p class="bottom-text">@ Meanwood Chamba Valley</p>
+</main>
+    <div class="main-section">
+        <h1 class="main-section-h1">
+            Main Program
+        </h1>
+            <div class="orange-underline"></div>
+                <ul class="blocks">
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Seminars</h2>
+                        <p>welcome to our free Seminars everyday</p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Lectures</h2>
+                        <p>Learn mordern farming techniques</p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Gov Programs</h2>
+                        <p>Get subsidized farming inputs</p>
+                    </li>
+                    <li><i class="fas fa-anchor"></i>
+                        <h2>Assistance</h2>
+                        <p>Access technical support</p>
+                    </li>
+                </ul>
+            <a href="#" class="see-more">See more</a>    
+     </div>
```

Comment:
- [ ] This issue has not been addressed yet. You should use `<section>` tag instead of `<div>` for your section containers to make your code more semantic. 
- [ ] Section still doesn't have the proper design

Please kindly fix these issues.

Reply:
https://www.loom.com/share/4edf87d810f64aa4b43649d604ff2884

Reply:
> https://www.loom.com/share/4edf87d810f64aa4b43649d604ff2884

Please add the video link to the pull request description so other code reviewers can notice it easily.

Reply:
- [ ] This section looks better now, but in the design template, this section has 5 articles. You should follow the composition given in the design

Reply:
This has been handled, thank you